WTWSMS/history/titles/d_orospeda.txt

```diff
@@ -1,7 +1,3 @@
-537.1.1={
-	holder=36555 # Athanagild Athanagilding   
-	liege = "k_visigoths"
-}
 567.1.1={
 	holder=36557 # Hildireths Sunedfriding     
 	liege = 0
```

Comment:
Wasn't Orospeda a Visigothic province?

Reply:
Yes, but it was independent in the mod, and a map says so but I haven't found any info about it being independent so I don't mind making the character a Visigothic vassal.

Reply:
No, it wasn't independent if the Visigothic king held it.

Reply:
That´s why I´m confused. A map shows Orospeda being independent but haven ´t found anything that supports that.

Reply:
But no. Orospeda was the rebell state but it was also a Visigothic province after and before that.

Reply:
But where does it say orospeda was the rebel state? There were rebels in Corduba and Athanagild rebelled in Seville.

Reply:
Orospeda had to be subdued by Leovigild in 577, according to Wickham, Chris, who mentions it in p.50  _Framing the Early Middle Ages: Europe and the Mediterranean, 400-800_, a book I posses.

Reply:
When did it gain independence?

Reply:
The book doesn't indicate that. Quoting p.94:

"After the Visigothic defeat by Clovis at Vouillé in 507, and their loss of all lands in Gaul except Septimania (Languedoc), the kingdom went into a two-generation political crisis. When Leovigild re-established royal authority in the 570s-580s he had to do it by conquest, for many parts of the peninsula where wholly independent: the Basques in the north, the Suevic kingdom in the north-east, the territory of Cantabria (probably based in the upper Ebro valley, and apparently ruled in Romanizing style by a senatus), the lands of the senior Aspidius (just south of Suevic Galicia), the city of Córdoba in the south, the unlocated Orospeda and Sabaria territories, and the Byzantine-controlled south-eastern coast, around Carthagena."

 (in the footnote Seville and Mérida are treated of with more temporary autonomy).

Reply:
OK. I'll add Orospeda back

Reply:
Just noticed that I had not actually deleted the Duchy of Orospeda, but it appears first in 567

Reply:
I think it should stay as it is in master, that should suffice.